src/diamond/handler/stats_d.py

```diff
@@ -152,7 +154,7 @@ def _connect(self):
             self.connection = statsd.StatsClient(
                 host=self.host,
                 port=self.port
-            )
+            ).pipeline()
```

Comment:
Is there documentation for this feature? Does this call need to be conditional for legacy support?